### DISCUSSION TITLE
[Projects Beta] Set allowed repos for a project

### DISCUSSION THREAD

**Discussion by nogginbox** [community/community#14225] [Projects and Issues]
When turning a draft issue to a repo issue you get to choose which repo to use.

I would like to be able to set which repos can be used for a project. Most of my projects are associated with only one repo.

This would prevent users from accidentally creating issues in the wrong project and those issues getting lost and forgotten.

**Comment by nogginbox** [comment]
This feature would compliment the feature request to [Support adding default repo for project](/github/feedback/discussions/8251).

**Comment by svaningelgem** [comment]
+1
